spec/factories/my_modules.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 FactoryBot.define do
   factory :my_module do
-    name 'My first module'
-    x 0
-    y 0
-    workflow_order 0
-    experiment { Experiment.first || create(:experiment_one) }
-    my_module_group { MyModuleGroup.first || create(:my_module_group) }
+    sequence(:name) { |n| "Task-#{n}" }
+    x { Faker::Number.between(1, 100) }
+    y { Faker::Number.between(1, 100) }
+    workflow_order { 0 }
```

Comment:
Actually, I did it in this way because we don't have model uniqueness validations for those 3 fields. 
For `workflow_order` I can increase with `.count` but I don't see a simple (not dirty) solution for `x`, `y` values. Until there is a random generator where we have 10k (x*y) values and we don't have model uniqueness validation, this should not be a problem or? 
Maybe the next task could be adding validators on models + and then handling this logic somewhere in model? 

LMK what do you think @Ducz0r

Reply:
Agreed. And if it's not validated on the model level, it's kind of far-fetched to be handling it here. Leave it for now.